azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/Group.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.graphrbac;
+
+import com.microsoft.azure.management.apigeneration.Beta;
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.graphrbac.implementation.ADGroupInner;
+import com.microsoft.azure.management.resources.fluentcore.model.HasInner;
+
+/**
+ * An immutable client-side representation of an Azure AD group.
+ */
+@Fluent(ContainerName = "/Microsoft.Azure.Management.Fluent.Graph.RBAC")
+@Beta
+public interface Group extends
+        HasInner<ADGroupInner> {
```

Comment:
are we able to put `HasManager<TManager>` on this as well? (like on most other models)